src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,12 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional
+                .ofNullable(user)
+                .filter(oc -> oc != null)
+                .filter(oc ->  oc.getAge() != null)
```

Comment:
```suggestion
                .filter(oc -> !Objects.isNull(oc.getAge()))
```
Objects 클래스를 찾아보고 활용해본다면 가독성을 높일수 있을것 같습니다. !